dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java

```diff
@@ -0,0 +1,75 @@
+package com.dotmarketing.startup.runonce;
```

Comment:
is it possible do a test?, maybe we can run the Task manually and check that the data base have the change that we expect